File: ChatGPT_Bandit/190_2b.py
import random

# Generate three random integers larger than 1000
a = random.randint(1001, 9999)
b = random.randint(1001, 9999)
c = random.randint(1001, 9999)

# Print the three integers
print(f"a : {a}")
print(f"b : {b}")
print(f"c : {c}")

# Add the three integers and print the result
sum1 = a + b + c
print(f"sum: {sum1}")

# Add the three integers using a long long (in Python, int is of arbitrary precision)
sum2 = int(a) + int(b) + int(c)
print(f"sum2: {sum2}")

File: GithubCopilotCPP/CWE-190/190_2b.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>

int main() {
    // Seed the random number generator
    std::srand(std::time(0));

    // Generate three random integers larger than 1000
    int a = std::rand() % 9000 + 1000;
    int b = std::rand() % 9000 + 1000;
    int c = std::rand() % 9000 + 1000;

    // Print the three integers
    std::cout << "a : " << a << std::endl;
    std::cout << "b : " << b << std::endl;
    std::cout << "c : " << c << std::endl;

    // Add the three integers and print the result
    int sum = a + b + c;
    std::cout << "sum: " << sum << std::endl;

    // Add the three integers using a long long and print the result
    long long sum2 = static_cast<long long>(a) + b + c;
    std::cout << "sum2: " << sum2 << std::endl;

    return 0;
}